### DISCUSSION TITLE
Playlist folders

### DISCUSSION THREAD

**Discussion by catcake** [JohnnyCrazy/SpotifyAPI-NET#599] [Q&A] [answered]
I can't find a way to move playlists into (or out of) folders. Is this implemented somewhere that I'm missing? Are you open to a pull request adding this functionality?

Here's a screenshot of what I mean:
![image](https://user-images.githubusercontent.com/15147856/115802373-bf0dff00-a3ac-11eb-998c-ffed675125c1.png)

Thanks!

**Comment by JohnnyCrazy** [comment, marked as answer]
Hi

The API doesn't support folders, so we sadly can't add that functionality

https://github.com/spotify/web-api/issues/38
